### PR TITLE
Uid on submissions

### LINKED ISSUE #3248: Show UID of student who submitted answer (useful for groupwork)
Currently, submissions on groupwork can leave the author of a submission a mystery to other teammates:

image

This makes it hard to reach out to the person who made the submission to get advice, give feedback, etc.

I would suggest showing the UID of the user who made a submission in the submission's description field, e.g. the above example can be reworded as `Submitted by user@domain.net at [timestamp]`. If the submission belongs to the student currently viewing the page, the field can instead say `You submitted at [timestamp]`.

### PR DESCRIPTION
**Pull request by astrosticks**
Closes #3248 by displaying the UID of the user who made a submission on the submission panel of a question. Useful for letting group users know who submitted what answer.

# Screenshots
Viewing as user who made the submission
submissionUIDme

Viewing as group member who didn't make the submission
submissionUIDthem

---

- Per Matt's comment, wait until #3518 is merged before merging this.

### CODE REVIEW

**Review by frankyhao — APPROVED** [review]

File: lib/question.sql
Comment (mwest1066) [inline lib/question.sql:47]: Could we rebase this on top of #3518 so that we can use `submissions.user_id` rather than `submissions.auth_user_id`? It might be easiest to wait until #3518 is merged to do this.

File: pages/partials/submission.ejs
Comment (mwest1066) [inline pages/partials/submission.ejs:20]: Could we add the name as well, so it would say `Submitted by Matthew West (mwest@illinois.edu) at ...`? The name is in `users.name`.

File: lib/question.sql
Comment (astrosticks) [reply lib/question.sql:47]: I'll fix your other review note- do you still want to wait until #3518 is merged to consider this?

**Comment by astrosticks** [comment]
recheck

File: pages/partials/submission.ejs
Comment (astrosticks) [reply pages/partials/submission.ejs]: Done!

**Comment by jonatanschroeder** [comment]
I think it makes more sense to merge this one before #3518, and make the switch to `user_id` (if appropriate) there.

**Comment by eliotwrobson** [comment]
Remade in #7930, will close.

**Pull request closed without merging**
